packages/vm/src/imports.rs

```diff
@@ -29,9 +29,9 @@ static MAX_LENGTH_DB_KEY: usize = 64 * KI;
 /// value for db_read/db_next (see DB_READ_VALUE_BUFFER_LENGTH in packages/std/src/imports.rs)
 static MAX_LENGTH_DB_VALUE: usize = 128 * KI;
 /// Typically 20 (Cosmos SDK, Ethereum) or 32 (Nano, Substrate)
-static MAX_LENGTH_CANONICAL_ADDRESS: usize = 32;
+const MAX_LENGTH_CANONICAL_ADDRESS: usize = 32;
 /// The maximum allowed size for bech32 (https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki#bech32)
-static MAX_LENGTH_HUMAN_ADDRESS: usize = 90;
+const MAX_LENGTH_HUMAN_ADDRESS: usize = 90;
```

Comment:
You can also change the other `static`s in this file, but it's out of scope so i'm approving anyways :)

Reply:
Yes that code is from before Ethan and me understood the difference. I try to keep the diff small to simplify cross branch merging.